src/cljs/hello_world/game.cljs

```diff
@@ -84,6 +84,7 @@
                                     send-puzzle-complete-fn!
                                     send-start-timer-fn!]}]
   (.setTo (.-scale (:play-button @util/game-state)) 0 0)
+  (.setTo (.-scale (:see-ranking-button @util/game-state)) 0 0)
```

Comment:
Base on this project, I made it a bit differently :) https://github.com/flyjwayur/aikakoneMatka-puzzle/pull/17/commits/8aae12803adf843103fdb5f0fc1363dfcc9a3440